src/controllers/HomeController.ts

```diff
@@ -0,0 +1,43 @@
+import { Get, HttpCode, JsonController, Res } from 'routing-controllers';
+import { OpenAPI } from 'routing-controllers-openapi';
+import { Response } from 'express';
+import statusCode from '../modules/statusCode';
+import util from '../modules/util';
+import message from '../modules/responseMessage';
+import { HomeService } from '../services/homeService';
+import { logger } from '../utils/Logger';
+
+@JsonController('/home')
+export class HomeController {
+  constructor(private homeService: HomeService) {}
+
+  @HttpCode(200)
+  @Get('')
+  @OpenAPI({
+    summary: '홈 데이터 조회',
+    description: '메인 화면 데이터 반환',
+    statusCode: '200',
+  })
+  public async getHomeData(@Res() res: Response): Promise<Response> {
```

Comment:
```suggestion
  public async getData(@Res() res: Response): Promise<Response> {
```
이미 home 컨트롤러이므로 변수명에 중복을 줄여도 좋을 것 같습니다

Reply:
7a39e911ace980af7ccc51ee9ac451addf7c24a6